openapi/components/schemas/Plans/FlexiblePlan.yaml

```
@@ -1,15 +1,36 @@
 type: object
 allOf:
   - $ref: ../Common/CommonPlan.yaml
-  - properties:
-      id:
-        readOnly: false
-        description: The ID of plan to modify with given settings.
-        allOf:
-          - $ref: ../ResourceId.yaml
-  - required:
-    - id
-    - name
-    - currency
-    - productId
-    - pricing
+properties:
+  id:
+    readOnly: false
+    description: The ID of plan to modify with given settings.
+    allOf:
+      - $ref: ../ResourceId.yaml
+  isActive:
+    type: boolean
+    description: Whether a plan is active.
+  invoiceTimeShift:
+    description: You can shift issue time and due time of invoices for this plan.
+    nullable: true
+    allOf:
+      - $ref: ../Invoices/InvoiceTimeShift.yaml
+  description:
+    type: string
+    description: The plan's description.
+  richDescription:
+    type: string
+    description: The plan's description enhanced.
```

nitpick: Good to add more detail here

```suggestion
    description: >-
         Rich-text description of the plan. This field accepts rich text formatting, 
         such as: bold, underline, italic, and hyperlinks.
```